homeassistant/components/melcloud/climate.py

```diff
@@ -317,16 +317,30 @@ async def async_turn_off(self) -> None:
     def min_temp(self) -> float:
         """Return the minimum temperature."""
         min_value = self._device.target_temperature_min
-        if min_value is not None:
+        max_value = self._device.target_temperature_max
+
+        if min_value is not None and max_value is not None and min_value <= max_value:
+            return min_value
+
+        if min_value is not None and min_value <= (
+            max_value if max_value is not None else DEFAULT_MAX_TEMP
+        ):
```

Comment:
So let's say `max_value` is not None, isn't this the same statement as above? Because then we would still compare `if min_value is not None and min_value <= max_value`